examples/bits.rs

```diff
@@ -77,7 +94,7 @@ fn main() {
         for byte in row {
             println!("  Byte {:02}:\n    Value: 0x{:02X}\n    Bits: {:08b}", row_num, byte, byte);
 
-            let bits = byte.bits::<LittleEndian>();
+            let bits: &BitSlice<_, _> = byte.bits::<LittleEndian>();
```

Comment:
I added some type declarations to demonstrate what is being produced by certain modifications. The `: &BitSlice` declarations give users a good start point to go look in the `bitvec` documentation if they want to learn more about how it's working.